sdk/all/src/test/java/io/opentelemetry/sdk/internal/SystemClockTest.java

```diff
@@ -22,7 +23,7 @@ void millisPrecision() {
     // If we test many times, we can be fairly sure we didn't just get lucky with having a rounded
     // result on a higher than expected precision timestamp.
     for (int i = 0; i < 100; i++) {
-      long now = SystemClock.getInstance().now();
+      long now = Clock.getDefault().now();
```

Comment:
it probably makes sense to keep calling `SystemClock.getInstance()` in the test of the SystemClock. :)